packages/protocol-contracts/contracts/ZetaConnector.non-eth.sol

```diff
@@ -17,6 +17,8 @@ interface ZetaToken is IERC20 {
 }
 
 contract ZetaConnectorNonEth is ZetaConnectorBase {
+    uint256 _maxSupply;
+
     constructor(
         address zetaTokenAddress_,
```

Comment:
I think we should add uint256 maxSupply_ as the last param of the constructor, otherwise we'll be depending on TSS to make it non-zero. WDYT @brewmaster012?

Reply:
who can change the maxSupply?

Reply:
There's a few options:
* We start with a max supply of max uint256 (no cap)
* We start with a max supply of 0 and 0 means no cap (I don't like it, it's counterintuitive and we may want 0 to actually be 0 in some cases)
* We add a parameter to set the initial max supply (requires many changes in the contracts, tests, and related functions)

In any case I think the cap should be able to be updated by TSS.